### DISCUSSION TITLE
Is there an easy way to mock functions or modules like with vitest?

### DISCUSSION THREAD

**Discussion by snake-py** [storybookjs/storybook#28668] [Help]
### Summary

I am using vite, msw and storybook. Inside my application there are multiple regiions which rely on some sort of date (today or now). These dates are mostly getting fetched by moment or the default Date api. Now I am trying to basically globally mock these functions.

e.g. 
- `Date.now()`
- `moment()`
- `moment().format()`

Is there an easy way to do that? 

I saw that I could basically somehow set the global time in the process, while this might work for this case other cases where I might want to mock anything else I am still at a loss.

**Comment by Marklb** [comment]
The docs discuss mocking modules: https://storybook.js.org/docs/writing-stories/mocking-data-and-modules/mocking-modules

There is an example specifically for the global `Date`: https://storybook.js.org/docs/writing-stories/mocking-data-and-modules/mocking-modules#setting-up-and-cleaning-up

**Comment by joebnb** [comment]
https://storybook.js.org/addons/storybook-addon-vite-mock this is more simple
